Cargo.toml

```diff
@@ -104,7 +104,7 @@ etcd-client = "0.12"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "b97efbf92a0bf9abcfa1d8fe0ffe8741a2e7309e" }
+greptime-proto = { git = "https://github.com/etolbakov/greptime-proto.git", rev = "1da8c466f9920ec85141bc673c544c627fe79416" }
```

Comment:
will update this back to `https://github.com/GreptimeTeam/greptime-proto.git` once https://github.com/GreptimeTeam/greptime-proto/pull/145 is merged

Reply:
This is merged now.